vllm/engine/metrics.py

```diff
@@ -35,9 +36,13 @@ class Metrics:
     details on limitations.
     """
     labelname_finish_reason = "finished_reason"
+    labelname_waiting_adapters = "waiting_adapters"
+    labelname_running_adapters = "running_adapters"
+    labelname_max_lora = "max_lora"
     _gauge_cls = prometheus_client.Gauge
     _counter_cls = prometheus_client.Counter
     _histogram_cls = prometheus_client.Histogram
+    _info_cls = prometheus_client.Info
```

Comment:
Unused?

Reply:
Thank you, i missed it. on side note I am trying to run format.sh, i used conda to install requirements-lint but i. still have mismatch of `Wrong clang-format version installed: 18.1.5 is required, not 18.1.8.`